.gitignore

```diff
@@ -23,4 +23,5 @@ open-machine-learning-jupyter-book/assignments/prerequisites/*
 */tmp/**
 */**/node_modules/*
 */node_modules/*
-awesome/database/node_modules
\ No newline at end of file
+awesome/database/node_modules

```

Comment:
Let's call it as .config.development.json. You may also need a prod one.

Reply:
done